bin/manage.py

```diff
@@ -0,0 +1,406 @@
+import fcntl
+import inspect
+import json
+import logging
+import os
+import socket
+import signal
+import struct
+import sys
+import time
+
+from functools import wraps
+
+import consul as pyconsul
+#import manta
+from pymongo import MongoClient
+from pymongo.errors import *
+
+consul = pyconsul.Consul(host=os.environ.get('CONSUL', 'consul'))
+logging.basicConfig(format='%(asctime)s %(levelname)s %(name)s %(message)s',
+                    stream=sys.stdout,
+                    level=logging.getLevelName(
+                        os.environ.get('LOG_LEVEL', 'INFO')))
+
+log = logging.getLogger('manage.py')
+
+def debug(fn):
+    """
+    Function/method decorator to trace calls via debug logging.
+    Is a pass-thru if we're not at LOG_LEVEL=DEBUG. Normally this
+    would have a lot of perf impact but this application doesn't
+    have significant throughput.
+    """
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        try:
+            # because we have concurrent processes running we want
+            # to tag each stack with an identifier for that process
+            arg = "[{}]".format(sys.argv[1])
+        except IndexError:
+            arg = "[pre_start]"
+        name = '{}{}{}'.format(arg, (len(inspect.stack()) * " "), fn.__name__)
+        log.debug('%s' % name)
+        out = apply(fn, args, kwargs)
+        log.debug('%s: %s', name, out)
+        return out
+    return wrapper
+
+def get_environ(key, default):
```

Comment:
In autopilotpattern/mysql we've updated this bit to include expanding variables from the environment so we may want to include that here: https://github.com/autopilotpattern/mysql/blob/08540e857282b331b63ecd77e52d6aa7106f2561/bin/manage.py#L60-L72